src/starkware/committee/committee/committee/committee.py

```diff
@@ -36,6 +37,8 @@
     "CommitteeObjectInfo",
     [("state_class", Type[StateBase]), ("tree_height", int), ("tree_class", Type[BinaryFactTree])],
 )
+next_batch_id_gauge = Gauge('committee_next_batch_id', 'Batch id just about to get processed')
```

Comment:
Gauge description ('Batch id just about to get processed') is not accurate. It's not necessarily 'just about' to get processed. At this stage it's waiting for the next batch, which could potentially take hours or even more.
A better description would be something like 'Id of next batch to get processed'.